### DISCUSSION TITLE
ATen/ATen.h: No such file or directory

### DISCUSSION THREAD

**Discussion by toygerse** [microsoft/vscode-cpptools#10099] [Q&A]
We want to import a header file in another directory,  when we import the header file, no error is reported, but an error is reported when compiling.
![%MH75U X`FDDFFDLA5OK$@H](https://user-images.githubusercontent.com/100067975/199228566-f04c7b0e-b603-47b1-9d96-21b8c49852e3.png)
We have set include path like this
 "includePath": [
                "${workspaceFolder}/**",
                "/usr/local/lib/python3.6/dist-packages/torch/include"
            ],

**Comment by sean-mcmanus** [comment]
The includePath property in c_cpp_properties.json doesn't apply to the command line you're using with nvcc. You need to add the `-I/usr/local/lib/python3.6/dist-packages/torch/include` or equivalent flag when compiling.
